src/components/Permissions/OpenappButton.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import CircleButton from 'cozy-ui/transpiled/react/CircleButton'
+import { useWebviewIntent } from 'cozy-intent'
+import { isFlagshipApp } from 'cozy-device-helper'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import OpenappIcon from 'cozy-ui/transpiled/react/Icons/Openapp'
+import AppLinker, { generateWebLink } from 'cozy-ui/transpiled/react/AppLinker'
+import { useClient } from 'cozy-client'
+
+export const OpenappButton = ({ type, matchingQueryResult, t }) => {
```

Comment:
let's provide only what we need : 

`type` and `matchingQueryResultData`

neither `t` nor `matchingQueryResult`